TACC/Core-Portal
# task/FP-1783: Editable welcome message

## Overview

Migrate welcome messages to custom messages, which can be editable via the admin panel.

## Related

* [FP-1783](https://jira.tacc.utexas.edu/browse/FP-1783)

## Testing

1. Checkout this branch and run `./manage.py migrate` in the `portal_django` container
2. Confirm welcome messages show up, can be dismissed, and state is persistent
3. Confirm you can edit the welcome messages from the admin panel, and the changes appear in the UI
